frontend/src/common/components/table/NoRecordsFound.tsx

```diff
@@ -7,7 +7,7 @@ export const NoRecordsFound = memo(() => {
     <div className="no-records-found">
       <div className="no-records-found__section no-records-found__section--img">
         <img
-          src="No_Data_Graphic.svg"
+          src="/No_Data_Graphic.svg"
```

Comment:
this wasn't coming through properly for me...but how was it working for everyone else up to now? weird

Reply:
do you still have problems with loading this image? or can I resolve this now?